apps/meteor/app/livechat/server/hooks/offlineMessageToChannel.ts

```diff
@@ -12,7 +12,7 @@ callbacks.add(
 			return data;
 		}
 
-		let channelName = settings.get('Livechat_OfflineMessageToChannel_channel_name');
+		let channelName = settings.get('Livechat_OfflineMessageToChannel_channel_name') as string;
```

Comment:
```suggestion
		let channelName = settings.get<string>('Livechat_OfflineMessageToChannel_channel_name');
```
We should avoid using `as Type`.